comparisons/serializers.py

```
@@ -81,4 +81,13 @@ class ComparisonDetailDataSerializer(serializers.Serializer):
-    data = ComparisonDetailDataSerializer()
\ No newline at end of file
+    data = ComparisonDetailDataSerializer()
+
+# 6. 기업 매치업 이름 수정용 
+class ComparisonNameUpdateSerializer(serializers.ModelSerializer):
+    # 모델의 title 필드를 API에서는 name이라는 이름으로 사용함
+    name = serializers.CharField(source='title')
+
+    class Meta:
+        model = Comparison
+        fields = ['name']
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)